app/views/admin/statuses/show.html.haml

```diff
@@ -47,7 +47,7 @@
 %h3= t('admin.statuses.history')
 
 %ol.history
-  - @status.edits.includes(:account, status: [:account]).each.with_index do |status_edit, i|
+  - @status.edits.reorder(nil).includes(:account, status: [:account]).find_each(order: :asc).with_index do |status_edit, i|
```

Comment:
The `edits` association has a default order. The `find_each` wants to override the default order. Fortunately the order on the association is `id: :asc` which is exactly what the `order: :asc` to `find_each` is going to recreate.

Possible refactor on this: either remove the default order on the association, and/or move this whole data setup into a controller helper and keep the query details out of the view.